content/en/docs/installation/compatibility.md

```diff
@@ -28,61 +28,6 @@ You can read more information on how to add firewall rules for the GKE control
 plane nodes in the [GKE
 docs](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#add_firewall_rules).
 
-Alternatively, you can read how to [disable the webhook
-component](#disabling-webhook).
 
-## Disabling Webhook
-
-If you are having issues with the webhook and cannot use it at this time, you
-can optionally disable the webhook altogether.
-
-Doing this may expose your cluster to miss-configuration problems that in some
-cases could cause cert-manager to stop working altogether (i.e. if invalid types
-are set for fields on cert-manager resources).
-
-How you disable the webhook depends on your deployment method.
-
-### With Helm
-
-The Helm chart exposes an option that can be used to disable the webhook.
-
-To do so with an existing installation, you can run:
-
-```bash
-$ helm upgrade cert-manager \
-     --reuse-values \
-     --set webhook.enabled=false
-```
-
-If you have not installed cert-manager yet, you can add the `--set
-webhook.enabled=false` to the `helm install` command used to install
-cert-manager.
-
-### With static manifests
-
-Because we cannot specify options when installing the static manifests to
-conditionally disable different components, we also ship a copy of the
-deployment files that do not include the webhook.
-
-Instead of installing with
-[`cert-manager.yaml`](https://github.com/jetstack/cert-manager/releases/download/v0.13.1/cert-manager.yaml)
-file, you should instead use the
-[`cert-manager-no-webhook.yaml`](https://github.com/jetstack/cert-manager/releases/download/v0.13.1/cert-manager-no-webhook.yaml)
-file located in the deploy directory.
-
-This is a destructive operation, as it will remove the
-`CustomResourceDefinition` resources, causing your configured `Issuers`,
-`Certificates` etc to be deleted.
-
-You should first [backup your configuration](../../tutorials/backup/) before
-running the following commands.
-
-To re-install cert-manager without the webhook, run:
-
-```bash
-$ kubectl delete -f https://github.com/jetstack/cert-manager/releases/download/v0.13.1/cert-manager.yaml
-$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.13.1/cert-manager-no-webhook.yaml
-```
-
-Once you have re-installed cert-manager, you should then [restore your
-configuration](../../tutorials/backup/).
+## Webhook
+Disabling the webhook is not supported anymore since `v0.14`.
```

Comment:
I think we may need to refine our docs around webhook troubleshooting - we currently have a FAQ page that links to this document: https://github.com/cert-manager/website/blob/9568ec9f8fd67f416297ff4acb3ec4942e7fe5c7/content/en/docs/faq/webhook.md

Neither of these two pages provide _that much_ debugging information. If anything, the `concepts/webhook.md` page provides the most in-depth info about what's going on.

Perhaps we should merge the FAQ page into this one, and extend it out to include information on how to debug issues with the webhook and/or explaining how the TLS assets are 'bootstrapped'? We're going to run into a lot more users with issues here over the coming weeks and having a single place to accumulate and communicate and advice would be great 😄

Reply:
Totally agreed! I made a seperate issue for this: https://github.com/cert-manager/website/issues/144